workflow-templates/im-deploy-windows-service.json

```
@@ -0,0 +1,5 @@
+{
+  "name": "Deploy - Deploy Windows Service",
+  "description": "Template that can be used to build and deploy a windows service",
```

It shouldn't be doing any building.  Different wording might be something like: 
```suggestion
  "description": "Template that can be used to deploy a windows service which has been configured for a specific environment",
```